RAQUASI88/src/WIN32/retroachievements.cpp

```diff
@@ -219,9 +221,12 @@ void RA_InitMemory()
     RA_ClearMemoryBanks();
     RA_InstallMemoryBank(bank_id++, MainRAMReader, MainRAMWriter, 0x10000);
 
-#if RA_ENABLE_VRAM
-    RA_InstallMemoryBank(bank_id++, HighSpeedRAMReader, HighSpeedRAMWriter, 0x1000);
-    RA_InstallMemoryBank(bank_id++, MainVRAMReader, MainVRAMWriter, 0x4000 * 4);
+#if RA_ENABLE_TVRAM
+    RA_InstallMemoryBank(bank_id++, TVRAMReader, TVRAMWriter, 0x1000);
+#endif
```

Comment:
You should have an `#elseif RA_ENABLE_GVRAM` here that defines an 4KB bank with a dummy reader that always returns 0 and doesn't do anything on write, or the GVRAM addresses may change depending on compilation flags. 

Because the banking feature was never fully implemented in the toolkit, the address of the first byte in a bank is just the address of the last byte of the previous bank plus one.

Reply:
Having compile flags for the registered memory seems generally dangerous. If an achievement is dependent on memory and then it's suddenly not available in a future release, the achievement will break.